addons/l10n_it/data/report_invoice.xml

```diff
@@ -2,8 +2,13 @@
 <odoo>
     <template id="report_invoice_document" inherit_id="account.report_invoice_document">
         <xpath expr="//p[@name='note']" position="replace">
-            <p t-if="o.move_type == 'out_invoice' and o.fiscal_position_id.note" name="note">
-                <span t-field="o.fiscal_position_id.note"/>
+            <p name="note">
+                <t t-if="o.move_type == 'out_invoice' and o.fiscal_position_id.note">
+                    <span t-field="o.fiscal_position_id.note"/>
+                </t>
+                <t t-else="">
+                    <span/>
+                </t>
```

Comment:
Is it use for xpath or it's useless?

Reply:
As I didn't know that there was an `xpath` targeting the `<p name="note">` there might be something targeting `//p [@name='note']/span`... who knows. 
Probably it's useless.
But I added it in v15 (tonight), so if I remove it, let's re-enable fw-ports

Reply:
Ok good for me!